test/html/expect/test_package+re/Nested/module-type-Y/index.html

```diff
@@ -27,21 +27,18 @@ <h1>
    <p>
     Some additional comments.
    </p>
+   <h2 id="type">
+    <a href="#type" class="anchor"></a>Type
+   </h2>
```

Comment:
This merge broke the HTML structure. This `h2` has nothing to do here, it should still be in `odoc-content`.

Reply:
Issue: https://github.com/ocaml/odoc/issues/631